docs/team/markusyeo.md

```diff
@@ -13,19 +13,25 @@ Given below are my contributions to the project.
 
 
 * **Enhancements implemented:**
-    * to be added soon
+  * **New Feature**: Changed default file storage format to be CSV
+    * Justification: As we deal with tabular data, we would like users to be able to easily view the data entered into `DengueHotspotTracker` offline. 
+  * **New Feature**: Added the ability to import and export data as a csv file
+    * Justification: This feature allows users to edit csv files to be able to import into `DengueHotspotTracker` and export out of.
```

Comment:
Maybe this could be phrased as allowing users to export files, edit them offline, then import them back into the DHT, which may be more convenient for sharing information with those without direct access to the DHT?

Reply:
Also nitpick but I think this feature might be better placed before the previous one! Because it seems to flow better to discuss the import/export features before talking about the CSV file format that it's imported / exported as.

Reply:
Okie thanks!

Reply:
I put the feature below since changing the file format to CSV seemed to come more intuitively before the import/export features. also just a draft hehe